backend/oracleobjectstorage/options.go

```diff
@@ -223,19 +219,6 @@ copied in chunks of this size.
 The minimum is 0 and the maximum is 5 GiB.`,
 		Default:  fs.SizeSuffix(maxSizeForCopy),
 		Advanced: true,
-	}, {
```

Comment:
In the s3 changes I left  `memory_pool_flush_time` and `memory_pool_use_mmap` but with `HideBoth` so we don't break existing scripts / config and just ignore these quietly.

Reply:
I introduced these flags recently in my previous PR that hasn't been released yet, so we are good wrt oracle backend that no one is using them.

Reply:
Great - I didn't spot that - thanks :-)